### PR TITLE
Fix form

### PR DESCRIPTION
**Pull request by drizko**
Add functionality to submit form + Remove console.log from keydown event

### CODE REVIEW

File: public/js/actions.js [hunk 1]
```diff
@@ -3,28 +3,36 @@ $(document).ready(function () {
...
+        columnForm = $("#columnSize").val();
+        rowForm = $("#rowSize").val();
+
+        window.game = new Game(columnForm, rowForm);
+        window.game.start();
+        appendData();
+    });
+
     window.game = new Game(columnForm, rowForm);
     window.game.start();
```
Comment (trizko) [inline public/js/actions.js]: you could now remove these since you call them on submit

File: public/js/actions.js [hunk 2]
```diff
@@ -3,28 +3,36 @@ $(document).ready(function () {
     var columnForm = $("#columnSize").val();
     var rowForm = $("#rowSize").val();
```
Comment (trizko) [inline public/js/actions.js]: you could remove these as well

**Comment by drizko** [comment]
@trizko updated

File: public/js/actions.js [hunk 3]
```diff
@@ -1,30 +1,31 @@
...
+    $("form").submit(function (event) {
+        event.preventDefault();
 
-    window.game = new Game(columnForm, rowForm);
-    window.game.start();
+        $(".table1").empty();
 
-    appendData();
+        columnForm = $("#columnSize").val();
+        rowForm = $("#rowSize").val();
```
Comment (trizko) [inline public/js/actions.js]: you need to declare these

**Comment by trizko** [comment]
:+1:

**Pull request merged**